### ISSUE TITLE
Nice but really slow

### ISSUE THREAD

**Issue by reagle** [liob/pandoc_reader#9]
Using pandoc generation (with citeproc) on my site goes from a few seconds to many minutes (still waiting). I wondering if there's a way to speed it up. If it freshly parses my 4MB yaml bibliography for each of ~400 articles, that's gonna take a while....

**Comment by chriskrycho** [comment]
@reagle, the solution I came up with is to generate smaller, article-specific bibliographies. You can see a few [here](https://github.com/chriskrycho/chriskrycho.com/tree/master/content/bib). Splitting those out also makes them more useful for any would-be readers (as I've made them available directly on the site via a link).

**Comment by reagle** [comment]
Yes, I do the same thing. I generate a custom, temporary bibliography for every document I convert.

**Comment by reagle** [comment]
Note, pandoc-citeproc was [recently improved](https://github.com/jgm/pandoc-citeproc/commit/36a50eaf7748f6e20cfa450fb84d34628eb6659b) to dramatically decrease the time used in processing large bibliographies so it might be much more feasible to use this plugin.
